protocols/virtualfund/virtualfund-single-hop_test.go

```diff
@@ -14,26 +14,28 @@ import (
 	"github.com/statechannels/go-nitro/types"
 )
 
-type actor struct {
-	address     types.Address
-	destination types.Destination
-	privateKey  []byte
-	role        uint
-}
+func TestSingleHopVirtualFund(t *testing.T) {
 
-func signState(s state.State, a actor) state.SignedState {
-	ss := state.NewSignedState(s)
-	sig, err := s.Sign(a.privateKey)
-	if err != nil {
-		panic(err)
+	type actor struct {
+		address     types.Address
+		destination types.Destination
+		privateKey  []byte
+		role        uint
 	}
-	err = ss.AddSignature(sig)
-	if err != nil {
-		panic(err)
+
+	signState := func(s state.State, a actor) state.SignedState {
```

Comment:
Not related to this PR but I think `signState` can be removed and we can use `ss.Sign` instead.